frontend/src/component/executiveDashboard/hooks/useFilteredFlagsSummary.ts

```diff
@@ -38,8 +42,14 @@ export const useFilteredFlagsSummary = (
             },
         );
 
+        const flagsPerUser = users?.total ? sum.total / users.total : 0;
+
         return {
             ...sum,
+            flagsPerUser:
+                Number.isNaN(flagsPerUser) || flagsPerUser === 0
```

Comment:
Why are we using "N/A" if flags per user is 0? If there are 0 flags, then 0 flags per user is an entirely valid value, isn't it?